local_planner/src/nodes/star_planner.cpp

```diff
@@ -33,8 +33,8 @@ void StarPlanner::setBoxSize(Box histogram_box_size) {
   histogram_box_size_ = histogram_box_size;
 }
 
-void StarPlanner::setGoal(geometry_msgs::Point goal) {
-  goal_ = goal;
+void StarPlanner::setGoal(geometry_msgs::PoseStamped goal) {
```

Comment:
are we passing by value on purpose here?

Reply:
good catch